packages/cli/src/platform/$.ts

```diff
@@ -0,0 +1,20 @@
+import { Command, Commands } from '@prisma/internals'
+
+import { EarlyAccessFlagError } from '../../../migrate/src/utils/flagErrors'
+
+export class $ implements Command {
```

Comment:
In general would it make sense to rename `$`  to be `root` or `RootCommand` wdyt?

Reply:
Sure could be. The symbol denotes that it's a namespace which is different than a command. Technically could be named anything including "root" so $ tries to make that separation between namespace and command more explicit.

But maybe $ is just counter productive too!